app/controllers/items_controller.rb

```diff
@@ -28,6 +28,13 @@ def show
         end
     end
     
+    def destroy
+        if Item.exists?(params[:id])
```

Comment:
Does this perform the same way as
```ruby
item = Item.find_by(params[:id])
item.delete
```
Also note that `delete` does not invoke callbacks, whereas `destroy` does...
I don't know if that will be useful for deleting an item. It might be useful...